plugins/assimp/module/vtkF3DAssimpImporter.cxx

```diff
@@ -592,7 +592,8 @@ class vtkF3DAssimpImporter::vtkInternals
     {
       // Work around for https://github.com/assimp/assimp/issues/4620
       this->Importer.SetPropertyBool(AI_CONFIG_IMPORT_FBX_PRESERVE_PIVOTS, false);
-      this->Scene = this->Importer.ReadFile(filePath, aiProcess_LimitBoneWeights);
+      this->Scene = this->Importer.ReadFile(
+        filePath, aiProcess_LimitBoneWeights | aiProcess_ValidateDataStructure);
```

Comment:
A few things I can read in the documentation of this flag:

> This post-processing step is not time-consuming. Its use is not compulsory, but recommended.

So I think it's a good idea to enable it.

> It is recommended that you capture Assimp's log output if you use this flag

That sounds like something we should do. We should transfer Assimp logs to F3D log so that all messages from Assimp are sent to F3D console.